netcode.c

```diff
@@ -4162,8 +4204,17 @@ void netcode_server_connect_client( struct netcode_server_t * server,
 
     char address_string[NETCODE_MAX_ADDRESS_STRING_LENGTH];
 
-    /* TODO: try to find a way to deliver match itd. */
-    if ( !skillz_add_client_to_match( server, 111, client_id ) )
+    /* TODO: try to find a way to deliver match id. */
+    int testId = 0;
+    if( server->num_connected_clients >= 3 )
+    {
+        testId = 222;
+    }
+    else
+    {
+        testId = 111;
+    }
+    if ( !skillz_add_client_to_match( server, testId, client_id, client_index ) )
```

Comment:
Pick a var naming convention standard and stick to it?

Reply:
Fixed.